plugins/modules/job_invocation.py

```diff
@@ -117,6 +117,7 @@
       time_span:
         description:
           - Distribute tasks over given number of seconds
+          - This is deprecated with foreman_remote_execution-11.0.0
```

Comment:
deprecated or removed? the PR you linked fully removes that parameter from the API